core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1093,10 +1096,15 @@ public GHResponse route( GHRequest request )
         QueryResult fromQResult = qResults.get(0);
 
         double weightLimit = request.getHints().getDouble("defaultWeightLimit", defaultWeightLimit);
+        int maxVisistedNodesForRequest = request.getHints().getInt("routing.max_visited_nodes", maxVisitedNodes);
```

Comment:
The `routing` in front is good as indeed the count is growing. Also please for now use `someThing` notation - I've created #682 to make everything more consistent or to be able to use both.